src/cloud-api-adaptor/pkg/adaptor/cloud/cloud.go

```diff
@@ -34,7 +35,10 @@ import (
 )
 
 const (
-	Version = "0.0.0"
+	SrcAuthfilePath = "/root/containers/auth.json"
+	AgentConfigPath = "/run/peerpod/agent-config.toml"
+	AuthFilePath    = "/run/peerpod/auth.json"
```

Comment:
nit: Should these vars be prefixed with `Def` ?

Reply:
I opted to not call them DefaultSomething if they are hardcoded values that cannot be overridden.